RetailCoder.VBE/Refactorings/MoveCloserToUsage/MoveCloserToUsageRefactoring.cs

```diff
@@ -135,9 +135,8 @@ private void UpdateOtherModules()
                 oldSelection = module.GetQualifiedSelection();
             }
 
-            var newTarget = _state.AllUserDeclarations.FirstOrDefault(
+            var newTarget = _state.DeclarationFinder.MatchName(_target.IdentifierName).FirstOrDefault(
```

Comment:
Doing this everywhere might possible give a perceptible performance boost with inspections, especially in larger projects.